src/shell/lisa_shell

```diff
@@ -277,9 +277,12 @@ fi
 
 function _lisa-test-usage {
 cat <<EOF
-Usage: lisa-test [args] FILE[:CLASS]
+Usage: lisa-test [--wload-iterations ITERATIONS] [args] FILE[:CLASS]
```

Comment:
The reason I went for `--wload-iterations` was because the thing that's repeated is each workload and not the test as a whole... but actually I guess that's pretty much the same thing so will change it.